hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/StoreImpl.java

```diff
@@ -173,6 +173,7 @@ public boolean hasAssociation(TokenRelationshipKey tokenRelationshipKey) {
 
     @Override
     public boolean hasApprovedForAll(Address ownerAddress, AccountID operatorId, TokenID tokenId) {
+        if (Address.ZERO.equals(ownerAddress)) return false;
```

Comment:
Standard is to always use braces. Please check your auto format settings.
```suggestion
        if (Address.ZERO.equals(ownerAddress)) {
            return false;
        }

```

Reply:
Fixed